.scrutinizer.yml

```diff
@@ -2,6 +2,9 @@ filter:
   excluded_paths:
     - tests/*
 build:
+  image: default-bionic
+  environment:
+      php: 8.1.2
```

Comment:
Not sure what this is -- we don't need it 👍 

Once removed I'll approve the PR.

```suggestion
```

Reply:
Oh I didn't see that Scrutinizer is now working with this -- thanks! 🙏